orientdb/src/main/java/org/hibernate/datastore/ogm/orientdb/OrientDBProperties.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.datastore.ogm.orientdb;
+
+import org.hibernate.ogm.cfg.OgmProperties;
+
+/**
+ * @author Sergey Chernolyas (sergey.chernolyas@gmail.com)
```

Comment:
Could you add your name in the author without the email?
In past we removed it almost everywhere because it causes some issues when generating the JavaDoc